views/auth/login.php

```diff
@@ -2,18 +2,20 @@
 
 declare(strict_types=1);
 
+use Yiisoft\Form\FormModelInterface;
+use Yiisoft\Form\Widget\Field;
+use Yiisoft\Form\Widget\Form;
 use Yiisoft\Html\Html;
 
 /**
  * @var \Yiisoft\View\WebView $this
  * @var \Yiisoft\Translator\TranslatorInterface $translator
  * @var \Yiisoft\Router\UrlGeneratorInterface $urlGenerator
  * @var string $csrf
+ * @var FormModelInterface $formModel
  */
 
-$this->setTitle($translator->translate('layout.login'));
-
-$error = $error ?? null;
+$this->setTitle(Html::encode($translator->translate('layout.login')));
```

Comment:
We should never escape input. Only output should be escaped.

So... https://github.com/yiisoft/demo/blob/master/views/layout/main.php#L46 there we should encode but not here.

Reply:
done.